deploy/environments/applications.yaml

```diff
@@ -150,7 +150,7 @@ spack:
     - reportinglib%intel
     - spatial-index
     - spykfunc
-    - steps+petsc
+    - steps+distmesh+petsc
```

Comment:
@tristan0x : `distmesh` is only with `develop` right? As we are deploying `3.6.0` as a default, `distmesh` won't be deployed anyway?

Reply:
3.6.0 is not affected by the +distmesh variant.
But spack CI seems to build the master branch of STEPS somehow, and the build of the master branch is broken without +distmesh  (I am fixing that).